src/frontend/jsonapi/index.md

```diff
@@ -20,7 +20,14 @@ The offered URLs to the resources depend on the application that hosts the Aimeo
     });
 
     promise.done( function( options ) {
-        var params = {};
+        var args = {};
+        var params;
```

Comment:
`params` must be initialized as object because of `params[options.meta.prefix] = args;`